wandb/sdk/wandb_init.py

```diff
@@ -449,18 +449,82 @@ def init(
     id=None,
     settings: Union[Settings, Dict[str, Any], None] = None,
 ) -> RunBase:
-    """Initialize a wandb Run.
+    """Initialize W&B
+    Spawns a new process to start or resume a run locally and communicate with a
+    wandb server. Should be called before any calls to wandb.log.
 
     Args:
-        entity: alias for team.
-        team: personal user or team to use for Run.
-        project: project name for the Run.
+        job_type (str, optional): The type of job running, defaults to 'train'
+        dir (str, optional): An absolute path to a directory where metadata will
+            be stored.
+        config (dict, argparse, or absl.flags, optional): The config parameters
```

Comment:
awesome - thanks!

Reply:
I rewrote the docs but do you think we should consider making that a separate parameter (config_file) for clarity? @raubitsj @vanpelt

Reply:
Yeah, that seems like a good idea.  `config_file` sounds good, are you volunteering to implement this?

Reply:
@harukatab would you be willing to change your implementation to use config_file instead of config? I think we should try to avoid overloading parameters.

Reply:
I made a thread in the #api-design channel for how we handle the config file argument. We might need to change the docs if we were to change the implementation so I think you can hold onto this config doc for now! And I'd be willing to make a change in implementation once we finalize the idea. Thank you Lukas!